lighthouse-core/audits/byte-efficiency/total-byte-weight.js

```diff
@@ -60,14 +68,10 @@ class TotalByteWeight extends ByteEfficiencyAudit {
       const totalCompletedRequests = results.length;
       results = results.sort((itemA, itemB) => itemB.totalBytes - itemA.totalBytes).slice(0, 10);
 
-      // Use the CDF of a log-normal distribution for scoring.
-      //   <= 1600KB: score≈1
```

Comment:
IMO they've limited value in their current form since they're inconsistently documenting percentiles and near the context.options rather than the defaultOptions declaration. I'd prefer to stick to the graphs and get better at interpreting the PODR and median.

alternatively I had an idea that we express the score in terms of the green/yellow transition and yellow/red transition and the function just finds the curve that fits (yellow/red is already median) but I def don't want to make that change in this PR :)

Reply:
agreed on inconsistent across files and yeah, they would need to move to live near defaultOptions (they could live in the jsdoc, for instance).

I don't feel particularly strongly about them, but I do see the appeal of not having to click through and hover over the graph to get a sense of scoring.

On the other other hand, this kind of thing should likely live in the docs (and mayyyyybe the report?) anyways :)

Reply:
Let's nuke